umap/static/umap/js/modules/schema.js

```diff
@@ -563,4 +563,9 @@ export const SCHEMA = {
     type: Object,
     impacts: ['data'],
   },
+
+  _reference_version: {
```

Comment:
On my understanding, this property does not stand in the json metadata, and I understand the schema as a (simple to be refined) representation of it.

Reply:
Because we now check that the properties are part of the schema before applying them in the updaters, the replication of the `_reference_version` field wasn't replicated.

As a solution, I added it to the schema. My way of thinking about the schema isn't only a representation of the JSON metadata, it's a way to tell what needs to be updated when a property is updated on the map / datalayer.

As such, some properties might not belong to the JSON files, but still need to trigger changes. I believe we'll need to set properties specific to the sync.

I see different ways to solve this:

- Let them in the schema here, and mark them with a specific thing, to tell they are sync-related.
- Create another schema, with these specific fields
- My favorite: special-case these in the updaters. After-all, it's part of the sync protocol, so it could be done there.

Reply:
Or we may actually put the reference_version in the json instead of in the headers ?

Reply:
Why not ! As this might cause some other problems down the line, is it okay to ship this as-is, and iterate on the subject in a later PR?